app/src/components/BCIDView.tsx

```diff
@@ -30,6 +29,8 @@ const trustedLSBCCredentialIssuerRe =
 const redirectUrlTemplate = 'bcwallet://bcsc/v1/dids/<did>'
 const notBeforeDateTimeAsString = '2022-11-21T17:00:00.000Z'
 const connectionDelayInMs = 3000
+// const invitationId = '6cc22b56-fd0c-4b78-a7e4-c60c2e80e034'
```

Comment:
Any chance this will be reused again?

Reply:
Yes, for the test environment so I want to remember it.